packages/dataviews/src/components/dataform/stories/index.story.tsx

```diff
@@ -106,20 +105,7 @@ const fields = [
 	{
 		id: 'sticky',
 		label: 'Sticky',
-		type: 'integer',
```

Comment:
I took the opportunity to refactor this story to use the proper `boolean` type. It highlights how the field type provides a default `Edit` function that just works. For now, it's not important what's the default edit function, this can be changed when/if need it later.